src/App.js

```diff
@@ -1,8 +1,13 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useGapiDispatch } from './hooks/GAPI';
 import { ThemeProvider, CSSReset } from '@chakra-ui/core';
 import Router from './router';
 
 function App() {
+    const { loadGapi } = useGapiDispatch();
+    useEffect(() => {
+        loadGapi();
+    }, []);
```

Comment:
Correct if I'm wrong, but shouldn't here be also the initial fetching of the dashboard books ?